1-JavaScript/Week3/homework/ex5-shoppingCartPure.js

```diff
@@ -16,7 +16,16 @@ it pure. Do the following:
 5. Confirm that you function passes the provided unit tests.
 ------------------------------------------------------------------------------*/
 // ! Function under test
-function addToShoppingCart(/* TODO parameter(s) go here */) {
+function addToShoppingCart(shoppingCartArray, groceryItem) {
+  const newShoppingCartArray = [...shoppingCartArray];
```

Comment:
@YuliiaHelievierova You can simplify your solution here as the following:
 ```suggestion
 return [...shoppingCart, groceryItem].slice(-3);
```
This one line will do the same thing that you are doing. You can merge `groceryItem`  & `shoppingCartArray` then do slice(-3)  that will make sure you only have 3 items

Reply:
Here's more details about es6 spread operator https://www.w3schools.com/react/react_es6_spread.asp